### ISSUE TITLE
Have the trader class support multiple algos

### ISSUE THREAD

**Issue by shershah010** [tfukaza/harvest#12]
- [ ] Replace the `self.algo` in the trader class with a list and replace the `set_algo` function with a list. 
- [ ] Update the trader `run` function to iterate through the algos and run each

**Comment by tfukaza** [comment]
This is pretty cool since I run multiple algorithms in parallel too.

**Comment by shershah010** [comment]
> BTW how would you handles cases where 2 algorithms buy at the same time?

I think for know the first algorithm buys first algo in the list will do the first buy and the second algo in the list does the second. We could later implement it so that if there are two buys the at the trader o broker level, they are converted into one buy.

**Comment by tfukaza** [comment]
But what if there is not enough buying power to buy in both algorithms?

**Comment by shershah010** [comment]
I think it should be all or nothing, kind to keep idempotent-ness of a transaction. So if there is not enough buying power to buy all the stocks from all the algos combined, there should not be a partial buy. I don't really know. Another problem is what errors should quit the whole run process and which should just give warnings. I think we should discuss this to get better error handling.

**Comment by tfukaza** [comment]
Yeah I think we should put the multi-algo idea on hold until we flesh out the details.

**Comment by tfukaza** [comment]
Actually, if we are going to add a web server, then this feature is actually needed since it's no longer as easy to run multiple instances of harvest (since there will be multiple servers running on different ports). I have removed the `won't fix` label.
